src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -193,13 +208,66 @@ public static boolean checkTree(@NotNull IBlockAccess world, BlockPos pos)
             return false;
         }
 
-        //Get base log, should already be base log
-        BlockPos basePos = getBaseLog(world, pos);
+        final BlockPos[] baseAndTOp = getBottomAndTopLog(world, pos, new LinkedList<>(), null, null);
```

Comment:
I think this should either return some sort of tuple object or set the value of this tree's top and base logs.